.gitignore

```diff
@@ -5,6 +5,12 @@
 
 #ignore backups
 *~
+.idea/**
+CMakeCache.txt
+**/CMakeFiles/**
+DartConfiguration.tcl
+test-symbols.map
```

Comment:
Where is this file from?

Reply:
Excluded it again, probably was autoadded by intellij